tika-core/src/main/resources/org/apache/tika/parser/external/tika-external-parsers.xml

```diff
@@ -61,4 +61,57 @@
        <match>\s*([A-Za-z0-9/ \(\)]+\S{1})\s+:\s+([A-Za-z0-9\(\)\[\] \:\-\.]+)\s*</match>
      </metadata>
   </parser>
+  <parser>
+     <check>
+       <command>sox --version</command>
+       <error-codes>126,127</error-codes>
+     </check>
+     <command>env FOO=${OUTPUT} sox --info ${INPUT}</command>
```

Comment:
The `${OUTPUT}` token tricks the ExternalParser to setting `outputFromStdOut` to false and thereby loads the stream into metadata items (`extractMetadata`) rather than writing contents to the xhtml stream (`extractOutput`).